packages/client/hmi-client/src/components/model/tera-parameters-metadata.vue

```diff
@@ -1,5 +1,49 @@
 <template>
-	<ul>
+	<template v-if="isStratified">
+		<ul>
+			<li
+				v-for="([baseParameterId, values], index) in collapsedParameters.entries()"
```

Comment:
It would be good if we can do a bit of preprocessing ahead of time so we can avoid complicated logic in the templates, as currently it is a bit hard to follow.

I would suggest we process `collapsedParameters.entries()` into something like 
```
[
  { baseParameterId, values, isVirtualParameter },
  ...
]
```

Then in the template we can simplify:

```
<li v-for="(obj, idx) in computedList">
  <!-- doesnt exist in model -->
  <template v-if="obj.isVirtual">
    <tera-parameter-metadata-entry ... >
  </template>
  <!-- exist in model -->
  <template v-else>
    <tera-parameter-metadata-entry ... >
  </template>
  <!-- children --->
  <template v-if="obj.values.length > 1">
    <tera-parameter-metadata-entry ... />
  </tempplate>
</li>
```

Reply:
applied to parameters, doing initials in another branch